src/components/element-editor/basic-details/basic-details.tsx

```diff
@@ -12,35 +9,26 @@ import {
 import { TFormFieldValues } from "src/schemas";
 import {
     FieldEditorAccordionItem,
-    MandatoryFieldBox,
-    StyledTextarea,
     ToggleWrapper,
     Wrapper,
 } from "./basic-details.styles";
+import { Description } from "./common/description";
+import { Id } from "./common/id";
+import { Label } from "./common/label";
+import { MandatoryField } from "./common/mandatory-field";
+import { Placeholder } from "./common/placeholder";
```

Comment:
- combine all imports from `commons/` folder
- append `Field` to each component name for clarity

```suggestion
import {
    TypeField,
    LabelField,
    MandatoryField,
    IdField,
    DescriptionField,
    PlaceholderField,
} from "./common";
```

Reply:
can we rename the files as well?

e.g. `type.tsx` to `type-field.tsx`